src/main.rs

```diff
@@ -0,0 +1,72 @@
+extern crate glob;
+extern crate libc;
+extern crate rubyfmt;
+
+use std::fs::{metadata, read_to_string, OpenOptions};
+use std::io::{self, Read, Write};
+use std::path::PathBuf;
+
+use glob::glob;
+
+fn rubyfmt_file(file_path: PathBuf) -> io::Result<()> {
+    let buffer = read_to_string(file_path.clone())?;
+    let res = rubyfmt::format_buffer(buffer);
+    let mut file = OpenOptions::new()
+        .write(true)
+        .open(file_path)
+        .expect("file");
+    write!(file, "{}", res)?;
```

Comment:
Same thing here, I'd consider having `format_buffer` take `&mut Write` and pass the file directly (probably with a `BufWriter`)